package.json

```diff
@@ -7,7 +7,8 @@
   "scripts": {
     "clean": "rimraf dist es tmp lib",
     "test": "jest",
-    "prepublish": "in-publish && npm run build || not-in-publish",
+    "prepublishOnly": "npm run build",
+    "prebuild": "npm run clean",
```

Comment:
It looks like we’ll be cleanin’ twice with this, since `npm run build` also runs `clean`?

Reply:
Whoops I meant to take that out of build. Will fix!